pep-0002.txt

```diff
@@ -2,196 +2,62 @@ PEP: 2
-library, the integrators will check with the head maintainers for all
-contributions, to see if there are any updates to be included in the
-next release.  The integrators will evaluate any such updates for
-issues like backwards compatibility and may require PEPs if the
-changes are deemed to be large.
-
-The head maintainers should take an active role in keeping up to date
-with the Python development process.  If a head maintainer is unable
-to function in this way, he or she should announce the intention to
-step down to the integrators and the rest of the maintainers, so that
-a replacement can step forward.  The integrators should at all times
-be capable of reaching the head maintainers by email.
-
-In the case where no head maintainer can be found (possibly because
-there are no maintainers left), the integrators will issue a call to
-the community at large asking for new maintainers to step forward.  If
-no one does, the integrators can decide to declare the contribution
-deprecated as described in :pep:`4`.
-
-
-Open issues
-===========
-
-There needs to be some procedure so that the integrators can always
-reach the maintainers (or at least the head maintainers). This could
-be accomplished by a mailing list to which all head maintainers should
-be subscribed (this could be python-dev). Another possibility, which
-may be useful in any case, is the maintenance of a list similar to
-that of the list of PEPs which lists all the contributions and their
-head maintainers with contact info.  This could in fact be part of the
-list of the PEPs, as a new contribution requires a PEP.  But since the
-authors/owners of a PEP introducing a new module may eventually be
-different from those who maintain it, this wouldn't resolve all issues
-yet.
-
-Should there be a list of what criteria integrators use for evaluating
-contributions?  (Source code but also things like documentation and a
-test suite, as well as such vague things like 'dependability of the
-maintainers'.)
-
-This relates to all the technical issues; check-in privileges, coding
-style requirements, documentation requirements, test suite
-requirements.  These are preferably part of another PEP.
-
-Should the current standard library be subdivided among maintainers?
-Many parts already have (informal) maintainers; it may be good to make
-this more explicit.
-
-Perhaps there is a better word for 'contribution'; the word
-'contribution' may not imply enough that the process (of development
-and maintenance) does not stop after the contribution is accepted and
-integrated into the library.
-
-Relationship to the mythical Catalog?
+Anything accepted into the standard library is expected to be
+primarily maintained there, within Python's development infrastructure.
+While some members of the development team may choose to maintain a
+backport of a module outside of the standard library, it is up to them
+to keep their external code in sync as appropriate.
+
 
 References
 ==========
```

Comment:
Sorry, forgot to include this in my initial review—

I can't quite reach it, but this can simply be removed after using a proper inline link instead of a footnote for the one reference within.